File: UI/jamfix_mobilna/jamfix_mobilna/lib/models/usluge.dart

import 'package:json_annotation/json_annotation.dart';

part 'usluge.g.dart';

@JsonSerializable()
class Usluge {
  int? uslugaId;
  String? imePrezime;
  DateTime? datum;
  String? brojRacuna;
  String? nazivPaketa;
  String? cijena;
  int? proizvodId;


  Usluge(
      {this.uslugaId,
      this.imePrezime,
      this.datum,
      this.brojRacuna,
      this.cijena,
      this.nazivPaketa,this.proizvodId});

  factory Usluge.fromJson(Map<String, dynamic> json) => _$UslugeFromJson(json);

  Map<String, dynamic> toJson() => _$UslugeToJson(this);
}

File: RS2/JamFix.Services/Database/Usluge.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace JamFix.Services.Database
{
    public class Usluge
    {
        [Key]
        public int UslugaId { get; set; }
        public string? ImePrezime { get; set; }
        public DateTime Datum { get; set; }
        public string BrojRacuna { get; set; }
        public string Cijena { get; set; }
        public string NazivPaketa { get; set; }
        public bool Placeno { get; set; }
        public int ProizvodId { get; set; }

        public virtual ICollection<UslugaStavke> UslugaStavke { get; set; }
    }
}
